Adyen/Analytics/Models/AdyenAnalytics.swift

```diff
@@ -32,7 +32,7 @@ public enum AnalyticsEventTarget: String, Encodable {
     case expiryDate = "expiry_date"
     case securityCode = "security_code"
     case holderName = "holder_name"
-    case dualBrand
```

Comment:
This enum is public and this is a breaking change. Should we keep the old one with the deprecation annotation instead?

Reply:
It is SPI, no issue.

Reply:
Yes, thanks, this explains why diff tool didn't highlight this change.